tests/test_ops/test_deform_conv.py

```diff
@@ -34,6 +34,18 @@
 gt_deform_weight_grad = [[[[3.62, 0.], [0.40, 0.18]]]]
 
 
+def cat2batch(batch_size=1, *args):
```

Comment:
We can simply use `np.repeat` to expand the dim of batch so this function can be left out.

```python
np.repeat(input, batch_size)
```